admin/services/pupil.service.js

```diff
@@ -48,6 +48,22 @@ const pupilService = {
     }
   },
   /**
+   * Fetch one pupil filtered by pupil id and school id
+   * @param pupilId
+   * @param schoolId
+   * @returns {Promise.<*>}
+   */
+  fetchOnePupil: async (pupilId, schoolId) => {
+    // TODO: Introduce integration tests
+    try {
+      return await Pupil
+      .findOne({'_id': pupilId, 'school': schoolId})
+      .exec()
+    } catch (error) {
+      throw new Error(error)
```

Comment:
There is no need for this try/catch block - suggest you remove.

Reply:
I'd rather keep it for now as it's the same pattern used for all the methods in the service. Maybe best to revise them all at once?

Reply:
@pris54 please can you do this change as part of the PR, as it's superfluous, and we will likely forget if its left as-is.

Reply:
This method has been replaced, the new one doesn't use a try/catch block however, I'm not refactoring the remaining methods of the service at this point.

Reply:
@pris54 its a 5 minute job, and i would like to see consistency across our components.  Please refactor them all.